src/main/java/use_case/food_management/add_to_inventory/AddToInventoryInteractor.java

```diff
@@ -32,32 +33,35 @@ public void execute(AddToInventoryInputData addToInventoryInputData) {
 
         // item in inventory already
         if (inventoryDataAccessObject.getInventory(addToInventoryInputData.getOwnerId())
-                .getItems().containsKey(addToInventoryInputData.getFoodName())) {
+                .getItems().containsKey(getFoodName(addToInventoryInputData.getStudySessionLength()))) {
+
+            final String foodName = getFoodName(addToInventoryInputData.getStudySessionLength());
 
             final int prevQuantity = inventoryDataAccessObject
                     .getInventory(addToInventoryInputData.getOwnerId())
-                    .getItems().get(addToInventoryInputData.getFoodName()).getQuantity();
+                    .getItems().get(foodName).getQuantity();
+
             final Inventory inventory = inventoryDataAccessObject.getInventory(addToInventoryInputData.getOwnerId());
-            foodItem = inventory.getItems().get(addToInventoryInputData.getFoodName());
+            foodItem = inventory.getItems().get(foodName);
```

Comment:
suggestion: have the food factory overload the create method and take the length of the study session as a parameter so that everything concerning creating the food would be dependent on one file - would follow open/close principle better since it would be easier to add more food items later